src/Microsoft.AspNetCore.Mvc.Razor.Extensions/Resources.resx

```diff
@@ -144,10 +162,22 @@
   <data name="NamespaceDirective_Description" xml:space="preserve">
     <value>Specify the base namespace for the current document.</value>
```

Comment:
@Eilon, should this be `Specify the base namespace for the page.` then?
FYI, https://github.com/aspnet/Razor/pull/1667#discussion_r136457691